ClosedXML/Excel/Ranges/IXLRangeBase.cs

```diff
@@ -267,5 +267,17 @@ public interface IXLRangeBase : IDisposable
         IXLConditionalFormat AddConditionalFormat();
 
         void Select();
+
+        /// <summary>
+        /// Get cropped range.
+        /// </summary>
+        IXLRangeBase Crop(IXLRangeBase crop);
```

Comment:
I think `IXLRangeBase Intersection(IXLRangeBase range);` is a better name for this method, or do I not understand its intent?